src/hrflow_connectors/connectors/hrflow/schemas.py

```diff
@@ -1,280 +1,13 @@
 import typing as t
 
+from hrflow.hrflow.schemas import (  # noqa: F401
+    GeneralEntitySchema,
+    HrFlowJob,
+    HrFlowProfile,
+)
```

Comment:
Reminder : HrFlow 4.0.1 has been deployed and now, to use the schemas, you must use `from hrflow.schemas import ...` instead of `from hrflow.hrflow.schemas import ...`.